src/webgpu/api/validation/encoding/cmds/compute_pass.spec.ts

```diff
@@ -189,7 +195,8 @@ g.test('indirect_dispatch_buffer,device_mismatch')
   .fn(async t => {
     const { mismatched } = t.params;
 
-    const pipeline = t.createNoOpComputePipeline();
+    const layout = t.device.createPipelineLayout({ bindGroupLayouts: [] });
+    const pipeline = t.createNoOpComputePipeline(layout);
```

Comment:
When I ran the tests that call createNoOpComputePipeline without a parameter, the tests failed with the below error. Though I'm not sure if calling createNoOpComputePipeline without a parameter was intended, it looks like the tests succeed if we pass a dummy pipeline layout to createNoOpComputePipeline. Please correct me if we keep the existing createNoOpComputePipeline invocation.

```
  - EXCEPTION: no overload matched for createComputePipeline:
    value is not object
    while converting member 'layout'
    TypeError: no overload matched for createComputePipeline:
    value is not object
    while converting member 'layout'
        at F.createNoOpComputePipeline (/home/gyuyoung/github/cts/out-node/webgpu/api/validation/validation_test.js:301:28)
        at RunCaseSpecific.fn (/home/gyuyoung/github/cts/out-node/webgpu/api/validation/encoding/cmds/compute_pass.spec.js:94:24)
```

Reply:
This sounds like a bug of the dawn-node bindings rejecting the string `'auto'` as a layout. This would be because we changed the WebGPU API from implicitly selecting a default layout, to requiring you to explicitly request it by saying `layout: 'auto'`.

If you agree this sounds like what is happening could you please file a bug? https://crbug.com/dawn/new

Reply:
ok, I just filed a bug there - https://bugs.chromium.org/p/dawn/issues/detail?id=1515. Then, do you think this PR is unnecessary, right?

Reply:
enga@ answered on the issue. According to enga@, Chromium doesn't support `auto` yet.

Reply:
Austin didn't realize the bug was filed against dawn-node. I replied there.

Yes, I think we should keep the current invocations and close this issue.

Reply:
ok, thanks for clarifying it. Closed this PR.